cookbooks/bach_opentsdb/attributes/default.rb

```diff
@@ -0,0 +1,34 @@
+#######################
+# OpenTSDB attributes #
+#######################
+
+### Configuration file
+default['bach_opentsdb']['network']['port'] = 4242
+default['bach_opentsdb']['network']['bind'] = nil
+default['bach_opentsdb']['network']['tcp_no_delay'] = 'true'
+default['bach_opentsdb']['network']['keep_alive'] = 'true'
+default['bach_opentsdb']['network']['reuse_address'] = 'true'
+default['bach_opentsdb']['network']['worker_threads'] = nil
+default['bach_opentsdb']['network']['async_io'] = 'true'
+default['bach_opentsdb']['http']['staticroot'] = '/usr/share/opentsdb/static/'
+default['bach_opentsdb']['http']['cachedir'] = '/tmp/opentsdb'
+default['bach_opentsdb']['core']['auto_create_metrics'] = 'false'
```

Comment:
It probably makes more sense to set this to `true` .  We don't manage a whitelist for our other monitoring system (graphite) anyway.